src/main/frontend/components/whiteboard.cljs

```diff
@@ -176,9 +176,9 @@
           has-checked? (not-empty checked-page-names)]
       [:<>
        [:h1.select-none.flex.items-center.whiteboard-dashboard-title.title
-        "All whiteboards"
-        [:span.opacity-50
-         (str " · " total-whiteboards)]
+        [:div "All whiteboards"
+         [:span.opacity-50
+          (str " · " total-whiteboards)]]
```

Comment:
Fixes the missing space before the dot
<img width="440" alt="CleanShot 2022-09-19 at 13 03 50@2x" src="https://user-images.githubusercontent.com/10744960/191713818-5cb6ad09-4842-440c-aee4-0372dba00293.png">